Add readme to protocol module (#23)
zeta-chain/zetachain: packages/protocol-contracts/README.md

# ZetaChain Protocol Contracts

This package includes:
* Contracts that interact directly with the protocol (Connector, Zeta token).
* Utilities and interfaces to make it easy to develop multichain dApps on ZetaChain (ZetaInteractor, ZetaInterfaces, ZetaTokenConsumer).

> ## Special addresses

Reviewer: ```suggestion
## Addresses
```

> - Tss Address: Collectively hold by Zeta blockchain validators.

Reviewer: ```suggestion
* TSS address: collectively held by Zeta blockchain validators.
```

> 
> - Tss Updater Address: This address will combine a Gnosis multi-sig and Open Zeppelin timelock to ensure transparency and improve security.

Reviewer: ```suggestion
* TSS Updater address: a Gnosis multi-sig that's under an OpenZeppelin TimeLock for security and transparency. Eventually, the ownership of this role will be delegated to the TSS address.
```
